packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc

```diff
@@ -52,7 +52,10 @@ class ConnectivityStreamHandler : public FlStreamHandler {
 
     ConnectionTypeCallback callback = [this](ConnectionType type) -> void {
       if (type != ConnectionType::kError) {
-        events_->Success(flutter::EncodableValue(ConnectionTypeToString(type)));
+        flutter::EncodableList encodedList;
```

Comment:
Suggest changing encodedList to encoded_list. Sorry,I didn't see that.

Reply:
Oh my fault, Thanks!